pkg/controller/servicediscovery/servicediscovery_controller.go

```diff
@@ -12,7 +12,7 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/types"
-	"sigs.k8s.io/controller-runtime/pkg/client"
+	. "sigs.k8s.io/controller-runtime/pkg/client"
```

Comment:
Minor thing but it's generally not considered best practice to use dot import (ginkgo/gomega dot imports in tests are ok).